src/functionalTest/groovy/com/autonomousapps/jvm/projects/IntegrationTestProject.groovy

```diff
@@ -133,7 +134,7 @@ final class IntegrationTestProject extends AbstractProject {
     emptyCompAdviceFor(':lib'),
     emptyCompAdviceFor(':core'),
     new ComprehensiveAdvice(':proj', [
-      Advice.ofChange(dependency(project('integrationTestImplementation', ':core')), 'api')
+      Advice.ofAdd(transitiveDependency(dependency(':core'), []), 'api')
```

Comment:
This expectation was "wrong" before. The 'proj.integrationTest' sources do depend on 'core'. So the dependency should not be moved from there.

Unless you assume that 'integrationTest' always inherits all 'implementation' dependencies and the rule is that you do not need to redeclare them then. But this may not be the case (it is not in this test case). There is no guarantee that the setup is like this and thus such assumptions should not be made.

If #451 is implemented, further thoughts could be put into this.